mriqc/workflows/functional.py

```diff
@@ -362,6 +393,14 @@ def individual_reports(settings, name='ReportsWorkflow'):
             (mosaic_spikes, mplots, [('out_file', 'in4')])
         ])
 
+    if settings.get('ica', False):
+        page_number = 4
+        if settings.get('fft_spikes_detector', False):
```

Comment:
They are, but one has to come in the report before the other and thus the "in" ports need adjustment.